scripts/us_census/acs5yr/subject_tables/common/resolve_geo_id.py

```diff
@@ -0,0 +1,48 @@
+"""
+Resolving geoIds given the GEOID string from data,census.gov
+"""
+
+## The map between summary level and FIPS code is based on
+# https://www.census.gov/programs-surveys/geography/guidance/geo-identifiers.html
+# A detailed note is available at https://mcdc.missouri.edu/geography/sumlevs/
+_SUMMARY_LEVELS_FIPS_CODE = {
+    '040': 2,
+    '050': 5,
+    '060': 10,
+    '140': 11,
+    '150': 12,
+    '160': 7,
+    '500': 4
```

Comment:
Comment on what these levels are?  Also, please provide an example for why we need to check the length besides the level too.

Can we also make this:
```
  '040': ('geoId/', 2),
  ...
  '860': ('zip/', 5),
```

And list all the supported levels

So code doesn't do any level-specific stuff.

Reply:
Thank you for the comments, I have updated the code based on them.

> Also, please provide an example for why we need to check the length besides the level too.

The initial idea I had was to check the GEOID for the summary level and also check if the length of the `fips_code` is the same as the expected length of FIPS code. The length check was more as a check for debugging the code. I have not seen the length check to be used in any of tests, and have removed it now.

Reply:
So should we get rid of it in the map too, and just have summary-level -> prefix?

Reply:
I have updated the map with `summary-level -> prefix`. For future reference, the expected `fips_code` length is moved to the comment along with the description of summary level. 

Something like this:
```python
    # State-level (fips_code length=2)
    '040': 'geoId/',
    # County-level (fips_code length=5)
    '050': 'geoId/',
    # State-County-County Subdivision (fips_code length=10)
    '060': 'geoId/',
```